notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/apns/models/payload/ApnsPayload.scala

```diff
@@ -4,22 +4,25 @@ import java.net.URI
 
 import _root_.models.NotificationType._
 import _root_.models._
-import com.gu.notifications.worker.delivery.ApnsPayload
+import com.gu.notifications.worker.delivery.{ApnsPayload => Payload}
+import com.gu.notifications.worker.delivery.utils.TimeToLive._
 import com.gu.notifications.worker.delivery.apns.models.payload.CustomProperty.Keys
 import com.gu.notifications.worker.delivery.apns.models.payload.PlatformUriTypes.{External, Item}
 import com.turo.pushy.apns.util.ApnsPayloadBuilder
 
+case class PayLoadAndTimeToLive(payLoad: String, timeToLive: Option[Long] = None)
```

Comment:
https://github.com/guardian/mobile-n10n/blob/master/notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/DeliveryPayload.scala#L6

Then you'd need some way of extracting the ttl from the `jsonString` value of `AprsPayload` to use here: 

https://github.com/guardian/mobile-n10n/blob/MSS-603/add-ttl-to-notification-payload/notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/apns/ApnsClient.scala#L55

(actually I have modified ApnsPayload, but i need to modify the case class, rather than the objject. Which is admittedly a bit fiddly)